index.html

```diff
@@ -24,18 +25,16 @@
           <h2>THE BEST FOOD BLOG ON THE WEB</h2>
           <nav class="right">
             <ul class="navlist">
-              <li class="soc_fb"></li>
-              <li class="soc_tw"></li>
-              <li class="soc_gg"></li>
-              <li class="soc_in"></li>
-              <li class="soc_fl"></li>
-              <li class="soc_pi"></li>
-              <li class="soc_rs"></li>
-              <li class="soc_em"></li>
+              <a href="facebook.com"><li class="soc_fb"></li></a>
+              <a href="twitter.com"><li class="soc_tw"></li></a>
+              <a href="googleplus.com"><li class="soc_gg"></li></a>
+              <a href="instagram.com"><li class="soc_in"></li></a>
+              <a href="flickr.com"><li class="soc_fl"></li></a>
+              <a href="pinterest.com"><li class="soc_pi"></li></a>
+              <a href="blogger.com"><li class="soc_rs"></li></a>
+              <a href="mailto:vanessa@delicious.com?Subject=I%20love%20your%20recipes" target="_top"><li class="soc_em"></li></a>
```

Comment:
Change these to
```
<li><a href="" alt=""><span class="icon"></span></a></li>
```
'span' elements find their use in icon font behavior by making them inline